chapters/01_introduction.tex

```diff
@@ -6,7 +6,7 @@ \chapter{Introduction}
 
 \begin{figure}
   \centering
-  \includegraphics{frog}
+  \imageKeepAspect{frog}
```

Comment:
Please keep the basic name `\includegraphics` in the sample. We will document various macros in the documentation. If you really want to highlight the difference, you can add another figure block. illustrating its benefits (you can reuse the image though).

Reply:
Reverted back to original